client/state/reader/posts/normalization-rules.js

```diff
@@ -42,11 +43,11 @@ export const
 	GALLERY_MIN_IMAGE_WIDTH = 350;
 
 function getCharacterCount( post ) {
-	if ( ! post || ! post.better_excerpt_no_html ) {
+	if ( ! post ) {
 		return 0;
 	}
 
-	return post.better_excerpt_no_html.length;
+	return stripTags( post.content ).length;
```

Comment:
Not sure if it's best to do this elsewhere in the post normalization rules, and store it on the post in `post.content_no_html` perhaps?